replication/binlogsyncer.go

```diff
@@ -25,10 +26,12 @@ type BinlogSyncer struct {
 	c        *client.Conn
 	serverID uint32
 
-	host     string
-	port     uint16
-	user     string
-	password string
+	// LocalHost is the name of you want to present to the MySQL master. If it is not set it will default to os.Hostname()
+	LocalHost string
```

Comment:
I think we should name it localHost or local?
lower case first. 

btw, where to set this?